src/AltinnCore/Runtime/Controllers/AuthenticationController.cs

```diff
@@ -36,21 +41,29 @@ public async Task<IActionResult> KeepAlive()
         {
             if (_settings.RuntimeMode != "AltinnStudio")
             {
-                HttpResponseMessage result = await _authentication.RefreshToken();
-                if (result.StatusCode != HttpStatusCode.OK)
-                {
-                    return StatusCode((int)result.StatusCode);
-                }
+                string token = await _authentication.RefreshToken();
 
-                string token = await result.Content.ReadAsStringAsync();
+                //if (result.StatusCode != HttpStatusCode.OK)
+                //{
+                //    return StatusCode((int)result.StatusCode);
+                //}
 
-                if (string.IsNullOrWhiteSpace(token))
+                //string token = await result.Content.ReadAsStringAsync();                
+
+                _logger.LogInformation($"Token value in keep alive:{token}");
+                CookieOptions runtimeCookieSetting = new CookieOptions
                 {
-                    HttpContext.Response.Cookies.Append(Common.Constants.General.RuntimeCookieName, token);
+                    Domain = _settings.HostName,                    
```

Comment:
Do we need to set the domain of the cookie both in startup and in this file? I figure we could do with only one of them.. The question then is, which one?

Reply:
yes, this doesnt take the setting from the startup. The startup settting is for the jwtcookieauthentication project. It doesnt impact this cookie